src/sentry/services/hybrid_cloud/__init__.py

```diff
@@ -34,10 +34,20 @@ class ApiProjectKey:
     dsn_public: str = ""
 
 
+@dataclass
+class ApiOrganizationMember:
+    # This can be null when the user is deleted.
+    user_id: Optional[int]
+    pass
+
+
 @dataclass
 class ApiOrganization:
     slug: str = ""
     id: int = -1
+    # exists iff the organization was queried with a user_id context, and that user_id
```

Comment:
Unsure if you're using the 'if and only if' shorthand here

```suggestion
    # exists if and only if the organization was queried with a user_id context, and that user_id
```

Reply:
Yes, I've found it pretty useful notation for explaining conditionals.  I can expand it if you prefer.